tests/testthat/test_makeNhoods.R

```diff
@@ -86,9 +86,15 @@ test_that("Incorrect input gives informative error", {
 
     # pass a graph but no reduced dimensions
     expect_error(makeNhoods(sim1.graph, d=30), "No reduced dimensions matrix provided")
+    
+    # pass a graph but no reduced dimensions - this should not throw and error for refinement_scheme = graph
+    expect_error(makeNhoods(sim1.graph, d=30, refined = TRUE, refinement_scheme = "graph", regexp = NA))
```

Comment:
What is the `regexp` argument here? This doesn't seem to be a valid argument to `makeNhoods`.

Reply:
It's an argument from expect_error that allows you to test that you don't get an error. I wanted to make sure you don't get an error even if you don't provide a reduced dimensions matrix when using refinement_scheme = "graph"